core/src/main/scala/caliban/wrappers/FieldMetrics.scala

```diff
@@ -56,16 +56,13 @@ object FieldMetrics {
     durationLabel: String = "graphql_fields_duration_seconds",
     buckets: Histogram.Boundaries = defaultBuckets,
     extraLabels: Set[MetricLabel] = Set.empty
-  ): Wrapper.EffectfulWrapper[Any] =
-    Wrapper.EffectfulWrapper(
-      for {
-        timings  <- ZIO.succeed(new AtomicReference(List.empty[Timing]))
-        failures <- ZIO.succeed(new AtomicReference(List.empty[String]))
-        clock    <- ZIO.clock
-        metrics   = new Metrics(totalLabel, durationLabel, buckets, extraLabels)
-      } yield overallWrapper(timings, failures, metrics) |+|
-        Unsafe.unsafe(implicit us => fieldWrapper(clock.unsafe.nanoTime(), timings, failures))
-    )
+  )(implicit clock: Clock = Clock.ClockLive): Wrapper[Any] =
```

Comment:
I quite prefer the old way, that let you customize the clock with `ZIO.withClock`, implicit parameter is a bit annoying.